docs/api/tracker-stores.rst

```diff
@@ -9,23 +8,23 @@ Tracker Stores
 
 .. edit-link::
 
-All conversations are stored within a `tracker store`.
-Rasa Core provides implementations for different store types out of the box.
-If you want to use another store, you can also build a custom tracker store by extending the `TrackerStore` class.
+All conversations are stored within a tracker store.
+Rasa Open Source provides implementations for different store types out of the box.
+If you want to use another store, you can also build a custom tracker store by extending the ``TrackerStore`` class.
 
 .. contents::
 
 InMemoryTrackerStore (default)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 :Description:
-    `InMemoryTrackerStore` is the default tracker store. It is used if no other tracker store is configured.
+    ``InMemoryTrackerStore`` is the default tracker store. It is used if no other tracker store is configured.
     It stores the conversation history in memory.
 
-    .. note:: As this store keeps all history in memory the entire history is lost if you restart Rasa Core.
+    .. note:: As this store keeps all history in memory, the entire history is lost if you restart the Rasa server.
```

Comment:
I'm not sure about this with regards to messaging, i feel like "the rasa server" is how it's referred to evvvverywhere. are we going to change that as well? bc then e.g. "the rasa services" in the docker compose setup are also "rasa open source services"?

Reply:
i know, but it's super confusing if we say Open Source somewhere, and not in other places... @TyDunn thoughts?

Reply:
I am also very much torn by this. I would say leave it as the Rasa server, since Karen called it the Rasa server in https://blog.rasa.com/how-to-migrate-your-assistant-to-rasa-x-the-easy-way/

Reply:
i'll leave it for now, but @akelad  maybe you want to bring it up and tag karen in the #rasa-messaging channel?